x-pack/plugins/code/index.ts

```diff
@@ -84,117 +47,9 @@ export const code = (kibana: any) =>
         isAdmin: Joi.boolean().default(true), // If we show the admin buttons
         disableScheduler: Joi.boolean().default(true), // Temp option to disable all schedulers.
         enableGlobalReference: Joi.boolean().default(false), // Global reference as optional feature for now
-        redirectToNode: Joi.string(),
+        codeNode: Joi.boolean().default(false),
```

Comment:
let's just use "enabled"

Reply:
Wouldn't disable the whole code plugin if it is disabled?